### ISSUE TITLE
Woocommerce Shop Page Empty

### ISSUE THREAD

**Issue by tushonline** [wallmanderco/elasticsearch-indexer#11]
When activated "Full" or "Search" only, the Woo "Shop" page is showing Empty. The headers & menu loads on it, but nothing shows up in content.

It works if I turn off indexing.

**Comment by mikaelmattsson** [comment]
Have you reindexed the site after changing the settings?

**Comment by tushonline** [comment]
Yes. Did a few tests. Doesn't work for /shop/ page.All other pages looked good. Using Divi theme. Also, my /shop/ page is not a a `shop_base` default in woocommerce. Just in case if that helps identifying,

**Comment by mikaelmattsson** [comment]
Could you enable the profiler and show me the output?
